events.html

```diff
@@ -29,32 +29,91 @@ <h3> TaxonWorks Together 2022 </h3>
   <li>Your Ideas: <a href="https://github.com/SpeciesFileGroup/taxonworks_together_2022">Please submit your input for events</a> at TaxonWorks Together 2022.</li>
 </ul>
 
-<h3> Weekly Digitization and TaxonWorks</h3>
+<h2> Regular Events by Month </h2>
+Welcome to our new open session schedule. All these events are on Wednesdays at the same time every week. The flexible agendas are set by those attending the meetings.
```

Comment:
I think you can remove "Welcome to our new open session schedule".  This only holds for some unknown time, and "open schedule" feels ambiguous.

Reply:
I've removed this sentence.